### ISSUE TITLE
Tradfri Bulbs reporting online when device offline

### ISSUE THREAD

**Issue by lonebaggie** [AlexxIT/XiaomiGateway3#134]
I have a number of Ikea bulbs reporting online when they have no power

The Hi Home app is correctly reporting  the bulbs offline,  Debug and HA are reporting them online 

```
2020-12-08 13:12:18  DEBUG    gateway3      192.168.1.50 | Setup Zigbee device {'did': 'lumi.90fd9ffffe748cba', 'mac': '0x90fd9ffffe748cba', 'model': 'ikea.light.led1536g5', 'type': 'zigbee', 'zb_ver': '1.2', 'init': {'light': 1, 'brightness': 100, 'color_temp': 250}, 'online': True}
2020-12-08 13:12:18  DEBUG    gateway3      192.168.1.50 | Setup Zigbee device {'did': 'lumi.90fd9ffffe8715c9', 'mac': '0x90fd9ffffe8715c9', 'model': 'ikea.light.led1536g5', 'type': 'zigbee', 'zb_ver': '1.2', 'init': {'light': 1, 'brightness': 100, 'color_temp': 250}, 'online': True}
2020-12-08 13:12:18  DEBUG    gateway3      192.168.1.50 | Setup Zigbee device {'did': 'lumi.90fd9ffffe89fcf4', 'mac': '0x90fd9ffffe89fcf4', 'model': 'ikea.light.led1536g5', 'type': 'zigbee', 'zb_ver': '1.2', 'init': {'light': 1, 'brightness': 100, 'color_temp': 250}, 'online': True}
2020-12-08 13:12:18  DEBUG    gateway3      192.168.1.50 | Setup Zigbee device {'did': 'lumi.90fd9ffffe8b10c2', 'mac': '0x90fd9ffffe8b10c2', 'model': 'ikea.light.led1536g5', 'type': 'zigbee', 'zb_ver': '1.2', 'init': {'light': 1, 'brightness': 100, 'color_temp': 250}, 'online': True}
2020-12-08 13:12:18  DEBUG    gateway3      192.168.1.50 | Setup Zigbee device {'did': 'lumi.90fd9ffffe8ef806', 'mac': '0x90fd9ffffe8ef806', 'model': 'ikea.light.led1536g5', 'type': 'zigbee', 'zb_ver': '1.2', 'init': {'light': 1, 'brightness': 100, 'color_temp': 250}, 'online': True}
2020-12-08 13:12:18  DEBUG    gateway3      192.168.1.50 | Setup Zigbee device {'did': 'lumi.90fd9ffffe92d75b', 'mac': '0x90fd9ffffe92d75b', 'model': 'ikea.light.led1536g5', 'type': 'zigbee', 'zb_ver': '1.2', 'init': {'light': 1, 'brightness': 100, 'color_temp': 250}, 'online': True}
2020-12-08 13:12:18  DEBUG    gateway3      192.168.1.50 | MQTT connected 
```

**Comment by AlexxIT** [comment]
Show retain logs lines. Was earlier.

**Comment by lonebaggie** [comment]
[gateway.txt](https://github.com/AlexxIT/XiaomiGateway3/files/5723298/gateway.txt)
Rebooted HA at 8am today. Logs are from re-boot.   Tradfri lights are physically off with no power. MI home reportng the tradfri bulbs as unavailable which is correct . HA is reporting tradfri  bulbs are  on   . If I turn them off in HA they switch back on . As soon as I return power to bulbs they report and work correctly. I suspect tradfri bulbs in HA   retain whatever state  they were   in  when power was turned off.

**Comment by lonebaggie** [comment]
Seems to be working in 0.11b4 bulbs now showing offline

**Comment by lonebaggie** [comment]
Update.  There seems to a sync issue . If  I reboot HA while the tradfri bulbs are unavailable , when HA reboots they report as on. Rebooting gateway has no effect.  Once I give power to the bulbs they re sync and work fine.

**Comment by lonebaggie** [comment]
Issue now resolved in v1.1b5 . Tradfri bulbs keep the correct state when HA is rebooted . Thankyou :)

**Comment by utrawoj** [comment]
Hi, how can i check if the tradfri bulb goes online when it was before power off? I use home assistant with node-red. Thx for your help, Sko
